src/consts/models.tsx

```diff
@@ -33,7 +33,7 @@ export const CONST = {
   pin: 'pin',
   defaultLanguage: 'en',
   maxAddressLength: 48,
-  tcVersionRequired: 2,
+  tcVersionRequired: 3,
```

Comment:
as far as I know, we did only one release with TC, so it should be version 2 now

Reply:
hmm must be a copy from emai-notification branch. not sure

Reply:
done any way for 2